### DISCUSSION TITLE
Alerting: ability to edit multiple alerts at once

### DISCUSSION THREAD

**Discussion by edin008** [grafana/grafana#49429] [Alerting]
Example: 100 alerts that need a change to one or more items from this list:
- time range
- labels
- evaluate every
- evaluate for
- folder
- group

Let's say I would like to add label "key=team, value=grafana" to 100 alerts.
Or that I would like to change time range of alert queries from "last 15 minutes" to "last 3 hours" to 100 alerts.

Editing each of 100 alerts would take a lot of time. I suggest implemening ability to edit multiple alerts at once.

**Comment by gtirloni** [comment]
I'm in the same situation. If anyone has a suggestion, I'd appreciate it.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
